abixen-platform-core/src/test/java/com/abixen/platform/core/infrastructure/configuration/PlatformTestSecurityConfiguration.java

```diff
@@ -14,21 +14,29 @@
 
 package com.abixen.platform.core.infrastructure.configuration;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Profile;
 import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
-
-import static com.abixen.platform.common.infrastructure.util.PlatformProfiles.DEV;
-import static com.abixen.platform.common.infrastructure.util.PlatformProfiles.DOCKER;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
 
 @Profile("test")
 @EnableWebSecurity
 @Configuration
 public class PlatformTestSecurityConfiguration extends WebSecurityConfigurerAdapter {
 
+    @Autowired
+    public void configureGlobal(AuthenticationManagerBuilder auth) throws Exception {
+        auth
```

Comment:
This formatting look weird.

Reply:
Better will by looks formating like
auth.userDetailsService(platformUserDetailsService())
       .passwordEncoder(bCryptPasswordEncoder());

Reply:
This is a standard of the formatting, see i.e. https://spring.io/blog/2013/07/03/spring-security-java-config-preview-web-security/